topdown/glob.go

```diff
@@ -31,6 +31,8 @@ func builtinGlobMatch(a, b, c ast.Value) (ast.Value, error) {
 		if len(delimiters) == 0 {
 			delimiters = []rune{'.'}
 		}
+	default:
```

Comment:
Sorry this isn't what I meant -- if it's not `null` and not an array, it might be something like
- `123`
- `true`
- `"foo"`
i.e. some other type of `Value`. We should not start to silently ignore that bad input, so let's return some sort of OperandError here, please.